Content.Shared/Teleportation/Components/HandTeleporterComponent.cs

```diff
@@ -20,6 +20,12 @@ public sealed partial class HandTeleporterComponent : Component
     [ViewVariables, DataField("secondPortal")]
     public EntityUid? SecondPortal = null;
 
+    /// <summary>
+    ///     Portals can't be placed on different grids?
+    /// </summary>
+    [DataField]
+    public bool PortalsFizzle = true;
```

Comment:
```suggestion
    public bool AllowPortalsOnDifferentGrids;
```
I was thinking more like this. Much clearer what it does. 
You'll need to reverse the logic too, of course.